cmd/moco-controller/cmd/run.go

```diff
@@ -113,6 +115,11 @@ func subMain(ns, addr string, port int) error {
 		return err
 	}
 
+	if err = (&mocov1beta2.MySQLCluster{}).SetupWebhookWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to setup webhook", "webhook", "MySQLCluster")
+		return err
+	}
+
 	if err = (&mocov1beta1.BackupPolicy{}).SetupWebhookWithManager(mgr); err != nil {
```

Comment:
With `matchPolicy=Eqiuvalent`, the webhook for v1beta2 can handle v1beta1 so this is not needed.

Reply:
I left the conversion webhook logic in v1beta1 because it is still there.